bin/reth/src/db/mod.rs

```diff
@@ -76,6 +77,10 @@ impl Command {
         match &self.command {
             // TODO: We'll need to add this on the DB trait.
             Subcommands::Stats { .. } => {
+                let mut stats_table = ComfyTable::new();
+                stats_table.load_preset(comfy_table::presets::ASCII_MARKDOWN);
+                stats_table.set_header(["Table Name", "# Entries", "Total Size (KB)"]);
```

Comment:
Can you add columns for branch, leaf and overflow pages respectively as well? :smile: